db/pf-schema-X.Y.Z.sql

```diff
@@ -1528,6 +1530,23 @@ CREATE TABLE `pki_revoked_certs` (
   KEY `revoked` (`revoked`)
 ) ENGINE=InnoDB DEFAULT CHARSET=latin1;
 
+--
+-- Table structure for table `bandwidth_accounting`
+--
+
+CREATE TABLE bandwidth_accounting (
+    tenant_id INT,
+    mac char(17) NOT NULL,
+    unique_session_id char(32),
```

Comment:
`mac` and `unique_session_id` should be INT columns to drastically increase index performance.

Re `mac`: strip chars ![a-f0-9] and store as either BINARY OR BIGINT
Re `unique_session_id`: BINARY(16) === CHAR(32) (as hex)
Re: `tenant_id`: Ideally UNSIGNED SMALLINT or UNSIGNED MEDIUMINT since we don't need negative ids. INT increases the index sizes beyond what's needed, thus unnecessarily increasing INDEX lookup performance.